### ISSUE TITLE
This is not an issue!

### ISSUE THREAD

**Issue by gilbertoferreira** [tizbac/proxmoxbackupclient_go#17]
Just wanna to say: "Thank you!"
I hope this can grow up 'till the moon.!

**Comment by Tacioandrade** [comment]
I would also like to congratulate you!!! Even though it is in alpha the tool is working perfectly!!!
I did some tests here and they are all working great! It will really be very useful, since some friends were doing tricks like sharing a folder in Samba and using the Debian client to back up the data, losing VSS support.
Much success to you not only in this project but also in other personal and professional projects.

**Comment by svendoering** [comment]
Hello @Tacioandrade How did you test the restore of the backed up files? Reason for this questions is that I am still not able to restore my "succesfull" backups. Looks great at the pbs, but when i try to access the files I get an error.

**Comment by gilbertoferreira** [comment]
> Hello @Tacioandrade How did you test the restore of the backed up files? Reason for this questions is that I am still not able to restore my "succesfull" backups. Looks great at the pbs, but when i try to access the files I get an error.

You need to open up the backup and then using the file restore level.

**Comment by Tacioandrade** [comment]
> Hello @Tacioandrade How did you test the restore of the backed up files? Reason for this questions is that I am still not able to restore my "succesfull" backups. Looks great at the pbs, but when i try to access the files I get an error.

I backed up one of the directories on a fileserver that had some XML, .conf and some executables and I was able to restore directly through the PBS web interface and restore the data without any problems. However, I didn't compile the source code, I got the Alpha 0.03 version which is here on Github itself.

**Comment by tizbac** [comment]
Hello, not being able to access files, usually indicates corrupt index, can
you try if that is the case with simpler backup ( less files ), too?

Il dom 25 feb 2024, 16:01 Tácio Andrade ***@***.***> ha
scritto:

**Comment by tizbac** [comment]
There was a problem with backup, please try version v0.0.3 thanks

**Comment by scheibling** [comment]
Yeah, I just wanna chime in here and say you've done an amazing job with this. We're running a couple of instances of Proxmox Backup Server, and the official backup client is quite a challenge to use in containerized environments and some other circumstances.

But this, this is amazing. I've run some tests locally stripping out most of the Windows code (and with that the requirement for CGO and other dependencies), which makes this able to run perfectly even in a distroless/scratch containers.

One thing, could it maybe make sense to separate the parts a bit and provide the core functionality as a go library, and then a separate repo (or just a separate installable package) for the windows and linux clients respectively? I can see a lot of applications for the basic library integrated into other tools for various backups to PBS

**Comment by Mraedis** [comment]
Loving this thing! :)

**Comment by tizbac** [comment]
FYI, on latest builds CGO dependency is gone , also linux version works flawlessy now :)
